sycl/include/CL/sycl/handler.hpp

```diff
@@ -1732,13 +1732,19 @@ class __SYCL_EXPORT handler {
 
   /// Copies data from one memory region to another, both pointed by
   /// USM pointers.
+  /// No operations is done if \param Count is zero. An exception is thrown
```

Comment:
Minor: typo in all of the 4 comments, "no operations is" -> "no operation is"/"no operations are"

Reply:
@v-klochkov, please, do not forget to fix the grammar in the comments.